mp4/stss.go

```diff
@@ -15,7 +15,6 @@ type StssBox struct {
 	Version      byte
 	Flags        uint32
 	SampleNumber []uint32
-	lookUp       map[uint32]bool // Used for optimization
```

Comment:
There's actually a thread safe map in the `sync` package.

https://golang.org/pkg/sync/#Map

Reply:
Thanks. That's true. Anyway, I don't want to spend the extra memory on a map in addition to the slice of sorted integers that is in the box.  This particular map code is an inheritance from https://github.com/jfbus/mp4 which I used as a start of this project.  The thread-safe map may be of interest if we change the implementation in the future to fully replace the slice of sorted numbers with a map.